manifests/claudie/dynamo/job.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-table-job
+  labels:
+    app: dynamo-create-table-job
+spec:
+  template:
+    metadata:
+      labels:
+        app: dynamo-create-table-job
+    spec:
+      restartPolicy: OnFailure
+      volumes:
+        - name: minio-configuration
+          projected:
+            sources:
+              - configMap:
+                  name: minio-cm
+      containers:
+        - name: awc-cli
+          image: amazon/aws-cli
+          args:
+            [
+              "dynamodb",
+              "create-table",
+              "--table-name",
+              "claudie",
+              "--attribute-definitions",
+              "AttributeName=LockID,AttributeType=S",
+              "--key-schema",
+              "AttributeName=LockID,KeyType=HASH",
+              "--provisioned-throughput",
+              "ReadCapacityUnits=1,WriteCapacityUnits=1",
+              "--endpoint-url",
+              "http://dynamo:8000",
```

Comment:
@MarioUhrik I wanted to use env vars for `endpoint-url` argument but didn't get it to work. I am unable to pass the variable in the command. I think i am using wrong syntax.

Reply:
If you don't get this to work, don't worry about it too much, it's not a dealbreaker at all

Reply:
But you can get it to work via setting an env variable from the configmap, and then referencing that env variable in the `args`

See https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#use-configmap-defined-environment-variables-in-pod-commands

Reply:
As suspected, i was using wrong syntax,`{}` instead of `()`. Thanks for guiding me here.